genesis/process/disabled/disabledBlockSizeComputationHandler.go

```diff
@@ -0,0 +1,32 @@
+package disabled
+
+// BlockSizeComputationHandler -
+type BlockSizeComputationHandler struct {
```

Comment:
Also here, might remove the "disabled" part of the filename. Also comments.

Reply:
done